Autopilot/PathManager/Inc/waypointManager.hpp

```diff
@@ -16,6 +16,7 @@
 
 #include <cstdint>
 #include <stdlib.h>
+#include "pathDatatypes.hpp"
```

Comment:
This doesen't look like it belongs here. Why did you decide to add it ?

Reply:
yup, doesn't belong. Will fix